raft/src/test/java/org/apache/kafka/raft/QuorumStateTest.java

```diff
@@ -945,9 +945,7 @@ public void testObserverUnattachedToFollower() throws IOException {
     }
 
     @Test
-    public void testInitializeWithCorruptedStore() throws IOException {
-        QuorumStateStore stateStore = Mockito.mock(QuorumStateStore.class);
-        Mockito.doThrow(IOException.class).when(stateStore).readElectionState();
```

Comment:
Why should we remove these 2 lines? After this PR, we should throw `UncheckedIOException` in corrupted store, is that right?

Reply:
You were right. I fixed it